ChangeLog.rst

```diff
@@ -30,11 +30,6 @@ deployment strategy for Elasticsearch `data` and `master` daemons
 
 :ghpull:`457` - update vendored Kubespray version
 
-:ghpull:`463` - install Mitogen_ as part of `make shell` and use the `mitogen`
```

Comment:
Would be nice to have a pointer in the revert commit message to the Mitogen issue which causes the need for this revert...